packages/apps/docs/src/docs/marmalade/what-is-marmalade.md

```diff
@@ -0,0 +1,180 @@
+  If the seller withdraws, the `pact` transfers the asset back to the seller's account from the escrow account.
+
+## Token policies
+
+The Marmalade token standard enables creators to specify **token policies** to control how their assets can be minted, sold, and transferred and how royalties are paid.
+For example, a token policy might specify that an asset can't be transferred directly to a buyer until a purchase is complete and the royalty distributed to the creator.
+
+Token policies are implemented as smart contracts based on the interfaces defined in the [`kip.token-policy-v2`](https://github.com/kadena-io/marmalade/blob/main/pact/kip/token-policy-v2.pact) standard.
+There are several built-in policies—called **concrete policies**—that are configured to handle the most common marketplace use cases. 
+For example, if you apply the `non-fungible-policy` to a token, the token is automatically configured to be minted only once. 
+
+You can apply built-in policies to any token or write custom policies based on the `kip.token-policy-v2` standard.
+You can also apply policies to **collections** of tokens to establish **marketplace communities** that share a common approach to token ownership or token sales. 
+
+For example, you can use token policies to define the following types of rules for creating, minting, burning, and transferring tokens:
+
+- You can create tokens with a **royalty** policy to specify a royalty rate as part of the token's metadata.
+
+- You can use a **mint** policy to specify whether a token is unique, part of a limited series, or a fully fungible coin.
+
+- You can specify a **burn** policy to prevent certain tokens from being destroyed.
+
+- You can define a **transfer** policy to restrict how tokens are transferred.
+
+## Preparing to mint a token
+
+Before you can mint a token on the blockchain, you must create the digital item and upload it to at least one accessible location such as InterPlanetary File System (IPFS), Arweave, or another content delivery or hosting service.
+
+After you have a uniform resource identifier (URI) for locating the digital item, you must also create a metadata file using the JSON schema to record the name, location, and other properties about the item to be minted.
+You can then create and mint the token for the digital item, generating an on-chain token identifier with a `t:` prefix. 
+The token identifier includes the hash for the metadata that describes the token to ensure that a digital item can be cryptographically verified to be the actual NFT. 
+
+Depending on how you apply or configure token policies, you might be the sole owner of the token you mint or share ownership with others. 
+If you're the owner of a token, you have the option to offer the token for sale or transfer the token to another owner. 
+
+## Token sales and trustless escrow
```

Comment:
In this section we don't exactly describe the sale contract's role. The sale can b e "quoted" sale or "non-quoted" sale, and when the seller uses the "quoted-sale", then they need to choose the "sale-contract". The "sale-contract" then has the ability to update the final price (such as the final price of the auction) or block withdraw until a condition is met. If the seller does not choose a quoted sale, sale-contract is not needed. Escrow part applies to any kind of sale, non-quoted or quoted. I think we can assume users are using quoted sale in this section, and briefly add the role of sale-contract.

Reply:
This is still kind of confusing to me. The source material maybe predates auction type sales. It seems like there are three types of sales:
Simple sale: I put up an offer with an asking price, a buyer pays the asking price (from your description, I think you are calling this a quoted sales, the terminology seems a little odd to me).
Conventional auction sale: with a low starting price and a well-defined window of time for bidders to act.
Dutch auction sale: with a high starting price and a step-down interval for the price to drop until someone acts (or the seller withdraws?)

Reply:
Yes, I think I requires some more context, I'll try to explain a bit better. I used the term "quoted sale" because a non-quoted sale is also available. 

In marmalade contract there are 3 types of sales 
1. sales that don't use quote 
2. sales that use quote without sale contract sale-type "" 
3. sales that use quote with sale contract i.e.) dutch-auction, conventional-auction

Case 1: 
 In this case, the sale's role is only the transfer of token from seller -> escrow -> buyer. 
There "could be" different custom logics that could be involved in token policies that custom it's own sale mechanism, but on the top level marmalade contract, we don't do anything. 

Case 2: 
Requires a "quote" from the seller, where the seller has to specify the fungible price of the sale. The buyer can process the "buy" by paying the quote price. Technically, it would be done by adding an empty sale-type of "" in the quote.

Case 3: 
Requires a "sale-contract" to be specificied with the quote. The sale-contract is granted the ability to "update" the quote price via its logic. 
Technical context:All sale contracts must provide `enforce-withdrawal`, `enforce-quote-update` that needs to pass at both "withdraw" step and "buy" step, giving the sale-contract authority to only allow buy's to be processed for the winning bids and winner accounts, or withdaws after only a certain condition. Conventional auction and dutch auction each store its logic. 

I think here we do describe the case 2 ,(quoted sales without sale contracts), and for an overview document it seems good enough.